backend/tests/integration/test_api_chat.py

```
@@ -310,3 +310,85 @@ def test_chat_does_not_duplicate_system_message(
+
+    def test_chat_streaming_requires_api_key(self, test_client):
+        """ストリーミングもAPI Keyが必要。"""
+        response = test_client.post(
+            "/v1/chat",
+            json={"messages": [{"role": "user", "content": "Hello"}], "stream": True},
+        )
+
+        assert response.status_code == 401
+
+    def test_chat_streaming_returns_sse_content_type(
+        self, test_client, mock_backend_config
```

Ruff (0.14.13)

Unused method argument: `mock_backend_config`

(ARG002)